docs/core/llm/index.md

```
@@ -282,3 +301,18 @@ The `llm.APIInstance` class has the following attributes:
 | `api_prompt`      | string  | Instructions for LLM on how to use the LLM tools. Required.                                      |
 | `llm_context`    | LLMContext | The context of the tool call. Required.                                                                 |
 | `tools`           | list[Tool] | The tools that are available in this API. Required.                                                     |
+| `custom_serializer` | Callable | Optional function to convert voluptuous schemas (for example selectors) into the JSON schema the LLM expects. Defaults to `None`. |
+
+## Exposing an API over MCP
+
+You do not need to do anything special to make your API available over the [Model Context Protocol (MCP)](https://modelcontextprotocol.io). Once the user sets up the [MCP Server integration](https://www.home-assistant.io/integrations/mcp_server/), every registered LLM API is automatically served over MCP.
+
+Each API is reachable at its own Streamable HTTP endpoint, addressed by its API ID:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)